### PR TITLE
gh-288  tx data loading

### PR DESCRIPTION
**Pull request by DmitryBespalov**
I could reuse the same transaction object

### CODE REVIEW

File: Multisig.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -113,6 +113,7 @@
 		0A29ECAA24C1EBFD00CE28FD /* Address+LocalizedStringKey.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0A29ECA924C1EBFD00CE28FD /* Address+LocalizedStringKey.swift */; };
 		0A339C5D24C1D16E00DAD84B /* Image+Blockies.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0A339C5C24C1D16E00DAD84B /* Image+Blockies.swift */; };
 		0A3F34DB2477E4E900E872AF /* KeyboardAdaptive.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0A3F34D82477E4E900E872AF /* KeyboardAdaptive.swift */; };
+		0A42C25424D0696D00ACA770 /* GetTransactionRequest.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0A42C25324D0696D00ACA770 /* GetTransactionRequest.swift */; };
```
Comment (MouazAlzahabi) [inline Multisig.xcodeproj/project.pbxproj]: Totally good, but I prefer to call it TransactionDetailsRequest instead

File: MultisigIntegrationTests/TokenRegistryIntegrationTests.swift [hunk 2]
```diff
@@ -18,40 +18,44 @@ class TokenRegistryIntegrationTests: XCTestCase {
...
-            tokens.append(self.registry[AddressRegistry.ether])
-            // pull from backend (AQER)
-            tokens.append(self.registry[cryptoKitties])
-            // same token, cached
-            tokens.append(self.registry[cryptoKitties])
-            // pull from blockchain (LINK)
-            tokens.append(self.registry[chainLinkNotInBackend])
-            // not a token -- should be recognized as ERC20 (this is a fallback logic)
-            tokens.append(self.registry[notAToken])
+            tokens.append(self.registry[.ether])
```
Comment (MouazAlzahabi) [inline MultisigIntegrationTests/TokenRegistryIntegrationTests.swift:46]: As you explicitly capture self, do we still need to use self. ?

**Review by MouazAlzahabi — APPROVED** [review]

File: MultisigIntegrationTests/TokenRegistryIntegrationTests.swift (hunk 2 above)
Comment (DmitryBespalov) [reply MultisigIntegrationTests/TokenRegistryIntegrationTests.swift:46]: yes, it's still not swift 5.2 :)

File: Multisig.xcodeproj/project.pbxproj (hunk 1 above)
Comment (DmitryBespalov) [reply Multisig.xcodeproj/project.pbxproj]: changed

**Pull request merged**